core/src/database/common.rs

```diff
@@ -918,6 +920,56 @@ impl Database {
 
         Ok((result.0 .0, result.1 .0, result.2, receipt))
     }
+
+    /// Sets Winternitz public keys for a watchtower.
+    #[tracing::instrument(skip(self, tx, winternitz_public_keys), err(level = tracing::Level::ERROR), ret(level = tracing::Level::TRACE))]
+    pub async fn save_winternitz_public_key(
```

Comment:
I think we might need to seperate saving these winternitz pubkeys with respect to operators but not sure.